docs/supported-services/route53zone.rst

```diff
@@ -0,0 +1,174 @@
+.. _route53zone:
+
+Route 53 Hosted Zone
+====================
+This document contains information about the Route 53 Hosted Zone service supported in Handel. This Handel service provisions a Route 53 Hosted Zone, in which you can create other DNS records.
+
+Service Limitations
+-------------------
+The following Route 53 features are not currently supported in this service:
+
+* Domain Name Registration
+
+Manual Steps
+------------
+If creating a public zone as a subdomain of another domain (like myapp.byu.edu), you must register it with your DNS provider.
+
+For BYU users, please refer to `this document <https://byuoit.atlassian.net/wiki/spaces/OAPP/pages/40075276/Routing+BYU+DNS+into+AWS>`_.
```

Comment:
We're trying to keep anything BYU specific out of Handel (other than the fact that we wrote it), so I'd like to remove this sentence. How about this section to be something more generic like the following:

```
If you are creating a public zone as a subdomain of another domain (like myapp.mydomain.com), you must register it with your DNS provider. 

If you are using Handel for your work at a company or organization of some kind, they likely have a process for registering these hosted zones with their DNS provider. Check with the networking groups in your organization to find out how you can do this.
```

That's just an example that could probably be improved of course, but we would like to keep anything BYU-special out of the library.